includes/class-wc-gateway-stripe.php

```diff
@@ -888,17 +888,17 @@ public function verify_intent_after_checkout( $order ) {
 			return;
 		}
 
-		if ( 'setup_intent' === $intent->object && 'succeeded' === $intent->status ) {
+		if ( 'setup_intent' === $intent->object && WC_Stripe_Payment_Intent_Status::SUCCESSFUL_STATUSES === $intent->status ) {
```

Comment:
This line isn't correct. It's using `WC_Stripe_Payment_Intent_Status::SUCCESSFUL_STATUSES` which is comparing an array to a string.

Reply:
Good catch, Matt! Thanks for pointing it out. Fixed it